File: kangaroo_business_app_sdk_web/lib/features/plugin_handler_list.dart
import 'package:kangaroo_business_app_sdk_web/base/plugin_channel_handler.dart';
import 'package:kangaroo_business_app_sdk_web/features/user_authentication/user_authentication_handler.dart';
import 'create_business_offer/create_business_offer_handler.dart';
import 'create_business_reward/create_business_reward_handler.dart';
import 'delete_offer_by_id/delete_offer_by_id_handler.dart';
import 'delete_recommendation_by_id/delete_recommendation_by_id_handler.dart';
import 'delete_reward_by_id/delete_reward_by_id_handler.dart';
import 'business_active_offers/business_active_offers_handler.dart';
import 'business_active_rewards/business_active_rewards_handler.dart';
import 'business_campaign_by_id/business_campaign_by_id_handler.dart';
import 'business_campaigns/business_campaigns_handler.dart';
import 'business_in_active_offers/business_in_active_offers_handler.dart';
import 'business_in_active_rewards/business_in_active_rewards_handler.dart';
import 'business_offer_by_id/business_offer_by_id_handler.dart';
import 'business_recommendations/business_recommendations_handler.dart';
import 'business_reward_by_id/business_reward_by_id_handler.dart';
import 'business_rules/business_rules_handler.dart';
import 'strings/strings_handler.dart';
import 'get_user_details/get_user_details_handler.dart';
import 'update_business_offer/update_business_offer_handler.dart';
import 'update_business_reward/update_business_reward_handler.dart';
import 'update_business_rules/update_business_rules_handler.dart';


List<PluginChannelHandler> pluginHandlerList = [
  CreateBusinessOfferHandler(),

  CreateBusinessRewardHandler(),

  DeleteOfferByIdHandler(),

  DeleteRecommendationByIdHandler(),

  DeleteRewardByIdHandler(),

  BusinessActiveOffersHandler(),

  BusinessActiveRewardsHandler(),

  BusinessCampaignByIdHandler(),

  BusinessCampaignsHandler(),

  BusinessInActiveOffersHandler(),

  BusinessInActiveRewardsHandler(),

  BusinessOfferByIdHandler(),

  BusinessRecommendationsHandler(),

  BusinessRewardByIdHandler(),

  BusinessRulesHandler(),

  StringsHandler(),

  GetUserDetailsHandler(),

  UpdateBusinessOfferHandler(),

  UpdateBusinessRewardHandler(),

  UpdateBusinessRulesHandler(),

  UserAuthenticationHandler()
];

File: kangaroo_business_app_sdk_android/android/src/main/kotlin/com/kangaroo/flutterplugin/android/features/PluginHandlerList.kt
package com.kangaroo.flutterplugin.android.features

import com.kangaroo.flutterplugin.android.base.PluginChannelHandler
import com.kangaroo.flutterplugin.android.features.userAuthentication.UserAuthenticationHandler
import com.kangaroo.flutterplugin.android.features.createBusinessOffer.CreateBusinessOfferHandler
import com.kangaroo.flutterplugin.android.features.createBusinessReward.CreateBusinessRewardHandler
import com.kangaroo.flutterplugin.android.features.deleteOfferById.DeleteOfferByIdHandler
import com.kangaroo.flutterplugin.android.features.deleteRecommendationById.DeleteRecommendationByIdHandler
import com.kangaroo.flutterplugin.android.features.deleteRewardById.DeleteRewardByIdHandler
import com.kangaroo.flutterplugin.android.features.businessActiveOffers.BusinessActiveOffersHandler
import com.kangaroo.flutterplugin.android.features.businessActiveRewards.BusinessActiveRewardsHandler
import com.kangaroo.flutterplugin.android.features.businessCampaignById.BusinessCampaignByIdHandler
import com.kangaroo.flutterplugin.android.features.businessCampaigns.BusinessCampaignsHandler
import com.kangaroo.flutterplugin.android.features.businessInActiveOffers.BusinessInActiveOffersHandler
import com.kangaroo.flutterplugin.android.features.businessInActiveRewards.BusinessInActiveRewardsHandler
import com.kangaroo.flutterplugin.android.features.businessOfferById.BusinessOfferByIdHandler
import com.kangaroo.flutterplugin.android.features.businessRecommendations.BusinessRecommendationsHandler
import com.kangaroo.flutterplugin.android.features.businessRewardById.BusinessRewardByIdHandler
import com.kangaroo.flutterplugin.android.features.businessRules.BusinessRulesHandler
import com.kangaroo.flutterplugin.android.features.strings.StringsHandler
import com.kangaroo.flutterplugin.android.features.getUserDetails.GetUserDetailsHandler
import com.kangaroo.flutterplugin.android.features.updateBusinessOffer.UpdateBusinessOfferHandler
import com.kangaroo.flutterplugin.android.features.updateBusinessReward.UpdateBusinessRewardHandler
import com.kangaroo.flutterplugin.android.features.updateBusinessRules.UpdateBusinessRulesHandler

val pluginHandlerList: List<PluginChannelHandler> = listOf(
    CreateBusinessOfferHandler(),

    CreateBusinessRewardHandler(),

    DeleteOfferByIdHandler(),

    DeleteRecommendationByIdHandler(),

    DeleteRewardByIdHandler(),

    BusinessActiveOffersHandler(),

    BusinessActiveRewardsHandler(),

    BusinessCampaignByIdHandler(),

    BusinessCampaignsHandler(),

    BusinessInActiveOffersHandler(),

    BusinessInActiveRewardsHandler(),

    BusinessOfferByIdHandler(),

    BusinessRecommendationsHandler(),

    BusinessRewardByIdHandler(),

    BusinessRulesHandler(),

    StringsHandler(),

    GetUserDetailsHandler(),

    UpdateBusinessOfferHandler(),

    UpdateBusinessRewardHandler(),

    UpdateBusinessRulesHandler(),

    UserAuthenticationHandler()
)
